catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/ImportCommand.java

```diff
@@ -102,12 +104,21 @@
   )
   boolean force = false;
 
+  @Option(
+    name = "--signature",
+    required = false,
+    aliases = {"-s"},
+    multiValued = false,
+    description =
+        "Provided absolute path for the digital signature to verify the integrity of the exported data. Required when the `--include-content` option is specified."
```

Comment:
❗️ I don't think there's an `--include-content` for this command, we may want to remove that portion of the description. 

✏️ Also should the wording of this be `Provide an absolute path..` (instead of `Provided absolute path`)